Index: wp-admin/includes/class-wp-list-table.php
===================================================================
--- wp-admin/includes/class-wp-list-table.php	(revision 35773)
+++ wp-admin/includes/class-wp-list-table.php	(working copy)
@@ -459,6 +459,10 @@
 		if ( empty( $this->_actions ) )
 			return;

+		if ( '2' === $two && 'plugins' === $this->screen->id ) {
+			return;
+		}
+
 		echo '<label for="bulk-action-selector-' . esc_attr( $which ) . '" class="screen-reader-text">' . __( 'Select bulk action' ) . '</label>';
 		echo '<select name="action' . $two . '" id="bulk-action-selector-' . esc_attr( $which ) . "\">\n";
 		echo '<option value="-1">' . __( 'Bulk Actions' ) . "</option>\n";
